lib/player_manager.rb

```diff
@@ -1,23 +1,31 @@
-require 'game_state'
+require 'ai_basic'
+require 'ai_minimax'
+require 'game_rules'
 class PlayerManager
 
   attr_reader :current_player
 
-  def initialize(game_io, player1, player2)
-      @player1 = player1
-      @player2 = player2
+  Player = Struct.new(:mark, :type)
+
+  def initialize()
+      @player1 = Player.new(:X, :Human)
```

Comment:
The way I the type works is that it either points to a symbol `:human` or to one of the Ai classes. It just seemed easier to have both initialize as human as default then have the driver program using the gem change decided if one should be a PC and what skill level it should be.